sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -667,4 +672,19 @@ export class StudentService extends RecordDataModelService<Student> {
       entityManager,
     );
   }
+
+  /**
+   * Get student PD status.
+   * @param pdVerified sfas pdVerified flag.
+   * @returns PD status.
+   */
+  private getPDStatus(pdVerified: boolean): DisabilityStatus {
+    if (pdVerified) {
+      return DisabilityStatus.PD;
+    }
+    if (pdVerified === false) {
+      return DisabilityStatus.Declined;
```

Comment:
For my info, will we be getting `declined` status from the current ATBC ( I remember we had back-and-forth discussions on this in our grooming)? 
if not, are we only considering the `declined` status only of SFAS entries?

Reply:
As of now, we are considering declined only for SFAS entries.

Reply:
Actually, addressing https://github.com/bcgov/SIMS/pull/2115/files#r1265791873 can eliminate this can avoid adding status to a new student